diagnostic.md

```diff
@@ -99,7 +134,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+The text color will be yellow and the font-family is sans-serif.
 <!-- your answer ends here -->
 
```

Comment:
font family will actually be cursive because the inline specificity of that styling is always the most powerful